konnect/login_service.go

```diff
@@ -53,7 +54,7 @@ func (s *AuthService) Login(ctx context.Context, email,
 	return authResponse, nil
 }
 
-func (s *AuthService) LoginV2(ctx context.Context, email,
+func (s *AuthService) basicAuth(ctx context.Context, email,
```

Comment:
```suggestion
func (s *AuthService) sessionAuth(ctx context.Context, email,
```

IT is technically not basic-auth.